docs/secure-communication-agents.asciidoc

```diff
@@ -24,21 +40,14 @@ are able to successfully use APM Server's API (except for RUM endpoints).
 
 To secure the communication between APM Agents and the APM Server with API keys:
 
-. <<ssl-setup-api,Enable SSL/TLS>>
+. <<ssl-setup,Enable SSL/TLS>>
 . <<configure-api-key,Enable and configure API keys>>
 . <<create-api-key,Create an API key with APM Server>>
 . <<set-api-key,Set the API key in your APM Agents>>
 
 NOTE: API Keys are not applicable for the RUM Agent,
 as there is no way to prevent them from being publicly exposed.
 
-[[ssl-setup-api]]
-[float]
-=== Enable SSL/TLS communication
-
-// Use the shared ssl short description
-include::./ssl-input-short.asciidoc[]
-
```

Comment:
Does it make sense to remove this step from this page? The list of steps above (for both API key/secret token) now includes the first step on an external page and the remaining three on this page. That seems confusing to me.

If you think it's more beneficial to explain encryption and auth on different pages, I think we should remove step 1 (`. <<ssl-setup,Enable SSL/TLS>>`) and instead add a prerequisite to the top of each page that explains that ssl/tls need to be enabled before continuing. It's mentioned in `secure-communication-agents.asciidoc`, but I worry people will miss that or just google directly to this page

Reply:
I _do_ think it makes sense - those are separate issues to tackle with regard to security. Potentially, a user can be good enough with SSL without additional authentication set up for certain environments.
Moreover, the contents of this section grew much larger (that's what this PR is about), so I think it doesn't make sense to embed twice anymore. 
But I see what you are saying - I will remove it from the steps and add it as a notice at the top of each page (API keys and Secret token)

Reply:
I ended up not removing the step - instead, I added a notice AND a step 1 that is not about setting SSL, but verifying it is setup, so I think it looks like a prerequisite enough now 🙂 .
LMK if you think I should remove one of them.